zulipterminal/helper.py

```diff
@@ -133,14 +132,16 @@ def set_count(id_list: List[int], controller: Any, new_count: int) -> None:
     all_msg = controller.view.home_button
     all_pm = controller.view.pm_button
     for id in id_list:
-        user_id = messages[id]['sender_id']
-
-        # If we sent this message, don't increase the count
-        if user_id == controller.model.user_id:
-            continue
-
         msg_type = messages[id]['type']
         add_to_counts = True
+        # FIXME no user_id for streams?
+        if msg_type != 'stream':
+            user_id = messages[id]['sender_id']
+
+            # If we sent this message, don't increase the count
+            if user_id == controller.model.user_id:
+                continue
```

Comment:
This is kind of hacky - we don't have `sender_id` for stream messages in `initial_data['unread_msgs']`
Though on close inspection this piece of code doesn't seem to be doing anything?
Refer to conversation on czo https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/Handling.20read.20events/near/845434